frontend/tests/controllers-test/authCtrl.test.js

```diff
@@ -322,7 +322,8 @@ describe('Unit tests for auth controller', function () {
                         }
                     });
                 } else {
-                    parameters.callback.onError();
+                    status = 400;
```

Comment:
Do we need to use a variable here?

Reply:
variable is remove and status is given directly now